### PR TITLE
Bump Android version and add more info to create_sdk_cipd_package.sh

### PR DESCRIPTION
**Pull request by blasten**
Bumps Android's build-tools,  platform-tools, and platforms packages.

Also, adds a bit more info to the script to help the next person doing the upgrade.

### CODE REVIEW

File: tools/create_sdk_cipd_package.sh
Comment (dnfield) [inline tools/create_sdk_cipd_package.sh:21]: Should we just create a script that does this?

Comment (blasten) [reply tools/create_sdk_cipd_package.sh:22]: it's durable, although a better approach is to create a script similar to https://github.com/google/android-emulator-container-scripts/blob/318c3c2b73c66f91a77aef90aa8ded71649901f5/emu/emu_downloads_menu.py that  parses the XML files with the assets, and then downloads the right stuff in the right place.

Given that this only occurs once a year, I'm not sure if we need this automation at the moment.

**Comment by blasten** [comment]
This change will need to wait for https://github.com/flutter/engine/pull/27376 as bumping the version requires an update to the buildroot repo as well.

**Comment by GaryQian** [comment]
The 31 SDK is still considered beta/first revision, do we have any guidelines/policies on what level of maturity is acceptable to upgrade Flutter to?

**Comment by blasten** [comment]
If a feature is available on Android S only (e.g bold text) and the API changes in a way that it requires a patch, we can roll a new SDK and apply the patch.  Similar to a Skia breaking change, etc.

This is unlikely, but I don’t think it should block the implementation of the Android S features.

**Review by GaryQian — APPROVED** [review]
Ok, that sgtm, this LGTM then, pending buildroot merge.

**Review by dnfield — APPROVED** [review]
LGTM as long as tests pass

**Comment by dnfield** [comment]
Looks like there are linting issues at the least

**Comment by blasten** [comment]
This is currently blocked on getting JDK 11 installed on the LUCI bots

**Comment by blasten** [comment]
The engine now depends on the JDK directly via gn dependencies.

**Pull request merged**
